
This knocks five minutes of the build time on my 2.4Ghz x86,
probably hours on really slow machines. There is no semantic
difference.

--- ../ghc-4.08.2/distrib/hc-build.orig	Sun Aug  3 17:20:59 2003
+++ ../ghc-4.08.2/distrib/hc-build	Sun Aug  3 17:21:05 2003
@@ -16,6 +16,7 @@
 GhcLibHcOpts = -O
 SRC_HAPPY_OPTS += -c
 GhcLibWays=
+SplitObjs=NO
 END
 
 # touch happy generated files; so that in non-bootstrapping mode for
